frontend/src/lib/components/Playlist/Playlist.tsx

```diff
@@ -221,7 +224,7 @@ const List = ({
     const initiallyOpenSections = sections.filter((s) => s.initiallyOpen).map((s) => s.key)
 
     return (
-        <div className="flex flex-col w-full bg-bg-light overflow-hidden h-full Playlist__list">
+        <div className="flex flex-col relative w-full bg-bg-light overflow-hidden h-full Playlist__list">
```

Comment:
i guess that this `relative` was really the fix but 🙈 

![css-blinds](https://github.com/user-attachments/assets/dd8ec0d0-eea6-413c-81ce-dd10216aa4a2)